xppenv

```diff
@@ -29,7 +29,8 @@ if [ -f "${HERE}/dev/hutch-python/bin/hutch-python" ]; then
 fi
 
 export PYTHONPATH="${HERE}:${HERE}/dev/devpath:${HERE}/../common/dev/devpath"
-source pcdsdaq_lib_setup
+#source pcdsdaq_lib_setup
+source /cds/group/pcds/pyps/apps/dev/pcdsdaq/bin/pcdsdaq_lib_setup
```

Comment:
@ZLLentz  do you remember what this is about? Do we still ned that?

Reply:
This was to work around a bug that was present in pcds-5.7.0 that caused the pcdsdaq_lib_setup script to be missing from the path

Starting at pcds-5.7.1 you can switch this back
See https://confluence.slac.stanford.edu/x/5uKDFg

Reply:
Thanks @ZLLentz ! Do you mind approving this so I can merge?